groovy/postInit/mod/Pyrotech.groovy

```diff
@@ -1076,6 +1128,8 @@ alloying_recipes = [
         ["Brass", 4, 200, ["Copper", 3, "Zinc", 1]],
         // SnFe
         ["TinAlloy", 2, 100, ["Iron", 1, "Tin", 1]],
+        // Potin
+        ["Potin", 9, 100, ["Bronze", 8, "Lead", 1]],
```

Comment:
isn't it better to have both options of mixing bronze + lead and mixing tin, copper and lead?

Reply:
it will result in recipe conflint